pd3/src/main/java/de/dema/pd3/persistence/ChatroomUser.java

```diff
@@ -0,0 +1,67 @@
+package de.dema.pd3.persistence;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+
+@Entity
+public class ChatroomUser implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+
+	@EmbeddedId
+	private ChatroomUserId id;
```

Comment:
Interessantes Beispiel, auch dass er @IdClass nicht angeben muss. Allerdings löst das Beispiel nicht das Problem, wie zwei ID-Spalten gemeinsam mit den Standard-Repositories verwendet werden können. In dem Beispiel wird für die betreffende Klasse gar kein Repository erzeugt.